src/types/widget.ts

```diff
@@ -262,10 +276,23 @@ export type Widget = {
   /**
    * Returns the render params to pass to the render function.
    */
-  getWidgetRenderState?(
+  getRenderState?(
     renderState: IndexRenderState,
-    widgetRenderStateOptions: InitOptions | RenderOptions
+    renderOptions: InitOptions | RenderOptions
   ): IndexRenderState;
+  /**
+   * Returns the render params to pass to the render function.
```

Comment:
Maybe be more precise about what this function returns, same for `getRenderState` so the differences between the two is clear for everyone.

Reply:
https://github.com/algolia/instantsearch.js/pull/4518/commits/6adc8b9c45eeea54f9391a7ffbfe2d7db1b6b5cf